src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -107,6 +115,14 @@ protected override void Dispose(bool disposing)
         /// <inheritdoc />
         protected override Task UpdateDisplayAsync(in RenderBatch batch)
         {
+            if (_prerrenderMode)
+            {
+                // Nothing to do in prerrender mode for right now.
+                // In the future we will capture all the serialized render batches and
+                // resend them to the client upon the initial reconnect.
+                return Task.CompletedTask;
+            }
+
```

Comment:
This all seems like a hack. non-blocking for this PR but we might to consider different designs for this.

Reply:
Fair

Reply:
I'll file a bug or it can be tagged along as part of @pranavkm reconnect werk

Reply:
Cool beans!